### ISSUE TITLE
Help: Deleting intervals?

### ISSUE THREAD

**Issue by gregcarpenter** [GoldenCheetah/GoldenCheetah#1955]
I cannot delete any intervals created with FIND INTERVALS.
I cannot see a DELETE command anywhere, except delete activity.
GC  ver:3.3 on WIN 7 64bit
Any suggestions?

thanks in advance.
Greg

**Comment by liversedge** [comment]
they are automatic, you can turn that off in preferences.

**Comment by liversedge** [comment]
apologies, rereading that -- you need to highlight the interval and use the interval sidebar hamburger menu.

if you cannot find either of those things watch the UI overview tutorial on the goldencheetah website

thanks

Mark

**Comment by gregcarpenter** [comment]
Hello Mark
Thanks for reply. I have watched the tutorial a few times and have seen intervals deleted.
I simply do not have a DELETE command in the INTERVAL 'hamburger' menu. I assume that's where it is.
I have highlighted 1 interval and many but I still cannot see a DELETE command anywhere.
Screen shot attached.
![gc - delete](https://cloud.githubusercontent.com/assets/19258718/15113054/581137cc-1635-11e6-8f98-91e23e1fa0d9.jpg)
.
